### ISSUE TITLE
[Item] Blazemails (Mark 2)

### ISSUE THREAD

**Issue by Kolkonut** [AngelArenaAllstars/suggestions#268]
_**This is a rewrite of the original idea! PM me if you would like the original.**_

All game item. Used on heroes that like to trade hits and fight/tank a lot (think of Axe, Bristleback, LC).

Crafted using a Radiance or any variations thereof and a Blademail. Can be upgraded with the same items as a Radiance (or its variations.)

**Passive Effect:** Any attacking unit has a 15% chance to cause <the effect of the Radiance used to craft it> for 1 second.
_Note 1: Heals will only affect the person with the Blazemail._
_Note 2: Does not cause blind, that would defeat the purpose of the item._

**Active Effect: Warflame:**
Cost 50 mana, lasts 5 seconds and has a cooldown of 45 seconds. (Ballpark numbers.)
Acts as a Blademail, reflecting x% of damage taken.
Also, triggers the normal effect of the Radiance used to craft the Blazemail, BUT the numbers are multiplied such that average heal/damage per second of the Blazemail is equal to the average heal/damage per second of the Radiance used to craft it.
_Note 3: During this time, the user is affected by True Strike i.e. attacks ON the user can NOT miss._

**Stats:** Provides some good armour and damage as well as the stats of the Radiance used to craft it.

Can be disassembled.

**Comment by VoidsKeeper** [comment]
Neat idea, @Kolkonut . To be honest,  I'd enjoy having such an item as an upgrade to blademail.
But care to explain what you meant by " However, the miss chance component only applies to units that are not attacking the user.."? So you mean that the radiance's miss chance doesn't apply when the item is active?

**Comment by Wujekklawy** [comment]
30s cooldown,  25 mana... OP as F.

**Comment by VoidsKeeper** [comment]
@Wujekklawy , I believe those numbers are just for show, so the basic mechanic is not affected. So it's a more or less  neat tank/initiator item

**Comment by Kolkonut** [comment]
@VoidsKeeper Consider three units, A,B and C. Let A have a Blazemail. A activates the Blazemail. 

Now consider B and C. Both of them are within the radius of the Blazemail effect. B targets C and attacks. These attacks have a X% miss chance.
Now let B attack A. These attacks will not miss.

This is to ensure that the effects of evasion and damage return do not negate each other.

Also,yes, the Radiance miss chance does not work unless the item is activated.

**Comment by Wujekklawy** [comment]
@Kolkonut or you just do it like centaur return,  no matter if you miss or not you will take damage

**Comment by Vafnar** [comment]
@Wujekklawy why not both? You could also do it like this:
On active, Blazemail gives X% chance to miss all _**allied**_ targets in the radius, but the player himself does not have a miss chance.

**Comment by Haganeko** [comment]
The passive is hard to follow. Does it get muted the the active is engaged?
I don't like the way the active blind works. Blademail punishes people for attacking you. They have counterplay, the item has a clear purpose. This just punishes people. It says "buy mkb or you lose"
Also, what is the blind miss %? Just a ballpark. Is it high like KotL or Tinker, or low like vanilla Radiance?

**Comment by Kolkonut** [comment]
@Vafnar that, good sir, is an excellent idea.

**Comment by Kolkonut** [comment]
@Haganeko around 15-20% perhaps.

**Comment by Kolkonut** [comment]
@All I must admit I prefer the combination of @Haganeko and @Wujekklawy suggestions. They feel better than my own idea.

**Comment by Haganeko** [comment]
I say combining a blademail effect for oneself with miss chance for attacking allies is... not on theme, I guess, is what I'm trying to say. I like the passive, and I like it getting muted on activation (because one wants people to hit them when blademail is active), but I am very indisposed towards it giving allies what is essentially evasion.

**Comment by VoidsKeeper** [comment]
@Haganeko , those could be two separate items , now that I think about it.

**Comment by Kolkonut** [comment]
@VoidsKeeper that might be better. This should probably only have the burn. Alternatively, perhaps crafting it with different variants of the Radiance change the active to Return+That Radiance?
Like the Lucience Blazemail heals allies, the Radiance Blazemail burns enemies etc.

**Comment by Kolkonut** [comment]
Is it alright if I rewrite the suggestion to be simpler to understand and integrate some of these great ideas?
@Haganeko @VoidsKeeper ?

**Comment by Haganeko** [comment]
Of course. Just remember Martyr's Mail is already a thing.

**Comment by VoidsKeeper** [comment]
Yeah, go ahead! :)

**Comment by Vafnar** [comment]
Let's get really complex... let's make the burn % amount, be damage dealt from the last use of the item divided by 10. For example, if the last time you used the active, you dealt 300 damage, then the next time you activate the effect, you deal 30 damage per second, creating a compound effect that is lost when the timing of the active is used wrong, punishing misplays harshly. This could only work against heroes though as I can see people abusing this against creeps. When the active is off however it goes back to baseline damage to that of normal Blazemail.

Or this can be totally ignored and maybe used for a different item, like an upgrade or something, idk, I'm **not** drunk at the time of writing this.

**Comment by Wujekklawy** [comment]
@Vafnar then you end up with item that deal 0 damage and there no way to change that.

**Comment by Kolkonut** [comment]
@Vafnar that would reward good play and stuff ,however, it would be rather finicky to use. It's definitely an interesting idea, perhaps suggest it as a separate item?

**Comment by Vafnar** [comment]
@Wujekklawy to counter this, have the starting damage be 10, if you hit all 5 heroes and 7 creeps, that is 120 damage and the next time you use it it will have 12 dps, if you hit all 5 heroes and 2 creeps, that is 84 dps, which means next time you deal 8.4 dps, and the cycle continues until the sweet sweet release of death that i yearn for at every waking moment

**Comment by Haganeko** [comment]
Still looks way too complicated. Closed.
